cmd/web/web.go

```diff
@@ -80,6 +103,24 @@ func serve(cmd *cobra.Command, args []string) {
 		cancel()
 	}()
 
+	app := initApp()
+	if err := app.Start(ctx); err != nil {
+		log.Fatal("Error while running the web application server: ", err)
+	}
+}
+
+func prune(cmd *cobra.Command, args []string) {
+	olderThan := viper.GetDuration("prune-events")
```

Comment:
I think we can user `.GetInt("older-than")`

Reply:
Should we do some sanity check here? 
What happens if the provided value is a negative value or similar? Maybe `viper` already has these kind of checks